src/client/pages/OfferNew/Introduction/HeroOfferDetails.tsx

```diff
@@ -1,75 +1,84 @@
 import React, { useState, useEffect } from 'react'
 import styled from '@emotion/styled'
 import { colorsV3 } from '@hedviginsurance/brand'
-import { Address } from 'data/graphql'
 import { OfferData } from 'pages/OfferNew/types'
 import { useTextKeys } from 'utils/textKeys'
-import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
-import { getHouseholdSize } from '../utils'
+import {
+  LARGE_SCREEN_MEDIA_QUERY,
+  MEDIUM_SCREEN_MEDIA_QUERY,
+} from 'utils/mediaQueries'
+import { getHouseholdSize, quoteDetailsHasAddress } from '../utils'
+import { getAddress } from '../Checkout/InsuranceSummaryDetails'
 
 type Props = {
   offerData: OfferData
 }
 
-type QuoteWithStreet = {
-  quoteDetails: { street: Address['street'] }
-}
-
-const Container = styled.div`
-  padding: 2rem 0 1rem;
+const Wrapper = styled.div`
+  padding: 2.5rem 0 1rem;
   color: ${colorsV3.white};
 
   ${LARGE_SCREEN_MEDIA_QUERY} {
     padding-top: 0;
   }
 `
-
-const Heading = styled.h1`
+const Headline = styled.h1`
   margin: 0;
   text-transform: uppercase;
-  font-size: 0.75rem;
+  font-size: 1rem;
+`
+const OfferInfoWrapper = styled.div`
+  padding: 1rem 0;
 
   ${LARGE_SCREEN_MEDIA_QUERY} {
-    font-size: 1rem;
+    padding-top: 0;
   }
 `
-
-const OfferInfo = styled.div`
+const NameAndCoInsured = styled.div`
   font-size: 2rem;
-  word-break: break-all;
 
-  ${LARGE_SCREEN_MEDIA_QUERY} {
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
     font-size: 3rem;
   }
 `
+const Address = styled.div`
+  font-size: 1.375rem;
+
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    font-size: 2.5rem;
+  }
+`
 
 export const HeroOfferDetails: React.FC<Props> = ({ offerData }) => {
   const [numberCoInsured, setNumberCoInsured] = useState<number | null>(null)
-  const [street, setStreet] = useState<Address['street'] | null>(null)
+  const [address, setAddress] = useState<string | null>(null)
```

Comment:
Do we really need state here? Can't we just derive it? Something like:

```
const { person, quotes } = offerData

const numberCoInsured = useMemo(() => {
  const householdSize = getHouseholdSize(quotes[0].quoteDetails)
  return householdSize - 1
}, [quotes[0].quoteDetails])

const address = useMemo(() => {
  const quoteWithAddress = quotes.find((quote) => quoteDetailsHasAddress(quote.quoteDetails))
  return quoteWithAddress ? getAddress(quoteWithAddress.quoteDetails) : null
}, [quotes])
```

Reply:
It might be smarter to do it this way but I disagree that this would simplify the code - to me this feels a lot more complicated actually.

Reply:
But it's probably unnecessary to have two states since they both will always change at the same time. Could do one state and an object instead.

Reply:
Or rather (sorry for this stream of consciousness lol) you're probably right about the state being unnecessary, but the answer, I believe, is not doing `useMemo`'s but just getting the values 🙃   
```typescript
const { person, quotes } = offerData
  const numberCoInsured = getHouseholdSize(quotes[0].quoteDetails) - 1

  const quoteWithAddress = quotes.find((quote) => {
    return quoteDetailsHasAddress(quote.quoteDetails)
  })
  const address = quoteWithAddress
    ? getAddress(quoteWithAddress.quoteDetails)
    : null
```

Reply:
Yeah, I don't think `useMemo` is necessary either :-)

Reply:
Fixed it now!